src/guides/v2.3/config-guide/prod/config-reference-most.md

```diff
@@ -276,6 +276,9 @@ Notify for Quantity Below | `cataloginventory/item_options/notify_stock_qty` | <
 Enable Qty Increments | `cataloginventory/item_options/enable_qty_increments` | <!-- ![Not EE-only]({{ site.baseurl }}/common/images/red-x.png) --> |
 Qty Increments | `cataloginventory/item_options/qty_increments` | <!-- ![Not EE-only]({{ site.baseurl }}/common/images/red-x.png) --> |
 Automatically Return Credit Memo Item to Stock | `cataloginventory/item_options/auto_return` | <!-- ![Not EE-only]({{ site.baseurl }}/common/images/red-x.png) --> |
+Provider | `cataloginventory/source_selection_distance_based/provider` | ![EE-only]({{ site.baseurl }}/common/images/cloud_ee.png) |
```

Comment:
Hi @sandipklevu. Thank you for your collaboration. 
I would ask you to double-check the fact that the mentioned settings are available in the Commerce edition only. They are part of the standard Inventory codebase (see the link in the comments) with no mention of being a part of `ScalableInventory` only. 

Thank you.

Reply:
Hi @rogyar Thanks for the suggestion. 
I have made the changes by removing the EE changes.